### PR TITLE
Tidy up a few minor things.

### PR DESCRIPTION
**Pull request by spxtr**
See individual commits.

### CODE REVIEW

File: prow/Makefile [hunk 1]
```diff
@@ -157,9 +148,3 @@ marque-service:
 	kubectl apply -f cluster/marque_service.yaml
 
 .PHONY: hook-image hook-deployment hook-service test-pr-image sinker-image sinker-deployment deck-image deck-deployment deck-service splice-image splice-deployment marque-image marque-deployment marque-service
-
-update-godeps:
```
Comment (ixdy) [inline prow/Makefile]: is this one actually dead?

File: prow/README.md [hunk 2]
```diff
@@ -10,19 +10,19 @@ currently comprises several related pieces that live in a GKE cluster.
 * `cmd/sinker` cleans up old jobs and pods.
 * `cmd/splice` regularly schedules batch jobs.
 * `cmd/deck` presents [a nice view](https://prow.k8s.io/) of recent jobs.
-* `cmd/phony` makes testing plugins easier.
+* `cmd/phony` sends fake webhooks.
+* `cmd/marque` is a production-ready letsencrypt certificate manager.
```
Comment (ixdy) [inline prow/README.md]: production-ready? brave words!

Comment (spxtr) [inline prow/README.md]: :)

File: prow/Makefile (hunk 1 above)
Comment (spxtr) [inline prow/Makefile]: Yep, since godeps live in the root of test-infra now this just won't work.

Comment (spxtr) [inline prow/Makefile]: (We should probably add an update-godeps script somewhere, but this code is just dead)

**Pull request merged**